package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "frontend-tech-test",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@lumx/icons": "^0.21.21",
+    "@lumx/react": "^0.21.21",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
+    "node-sass": "^4.14.0",
```

Comment:
Could we move this to `devDependencies` ? I know that CRA adds them to dependencies (I do not know why) but I see them more as `devDependencies` no?